docs/File-Formats.md

```diff
@@ -581,6 +581,10 @@ The MAF format recognized by the portal (excluding the annotation columns alread
 34. **t_ref_count (Optional)**: Reference allele count (tumor).
 35. **n_alt_count (Optional)**: Variant allele count (normal).
 36. **n_ref_count (Optional)**: Reference allele count (normal).
+37. **cBioPortal_binary_filter (Optional)**: "PUTATIVE_PASSENGER" or "PUTATIVE_DRIVER".
+38. **cBioPortal_binary_filter_annotation (Optional)**: field to give more information about the cBioPortal_binary_filter value (limited to 80 characters). This field can only be present if the cBioPortal_binary_filter is also present in the MAF file.
+39. **cBioPortal_multi_filter (Optional)**: free label that marks the mutation as a putative driver (limited to 20 characters).
+40. **cBioPortal_multi_filter_annotation (Optional)**: field to give more information about the cBioPortal_multi_filter value (limited to 80 characters). This field can only be present if the cBioPortal_multi_filter is also present in the MAF file.
```

Comment:
BioPortal_binary_filter does not fully describe the contents. What about:
- cbp_driver_passenger
- cbp_driver_passenger_annotation
- cbp_driver_class
- cbp_driver_class_annotation

The short name in column is also done in GDC MAF: https://docs.gdc.cancer.gov/Data/File_Formats/MAF_Format/ . The multi annotations are all drivers ‘tiers’ so calling this 'driver_class' makes sense I think. Another option would be:
- cbp_custom_driver_class
- cbp_custom_driver_class_annotation

If you agree with renaming the columns, it would also be nice if the variable names are improved